go.mod

```diff
@@ -1,4 +1,4 @@
-module github.com/coinbase/rosetta-bitcoin
```

Comment:
I don't think renaming the package is as easy as changing this, as the old package name is referened in lots of places (see [main.go](https://github.com/rosetta-dogecoin/rosetta-dogecoin/blob/main/main.go#L31))

Reply:
> You might want to squash some of those commits with rebase 😅

Good call, sorry I'm rusty.  Should I add cancel this rebase and resubmit?

Reply:
No it's okay! After you've rebased locally just force-push to this branch and the PR will still be here

Reply:
> No it's okay! After you've rebased locally just force-push to this branch and the PR will still be here

Ok learning moment, I git rebase -i 421cd95bdfea2079a412e4f3f06855e0d43ad600 to my last SHA and am wondering why it adds changes to all files +11,777 -11,777.  I understand it flattens the repo to reflect one commit but did I do it right?